client/src/components/portfolio/portfolio.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import PositionTable from './positionTable.jsx';
+import PortfolioChart from './portfolioChart.jsx';
+import AllocationChart from './allocationChart.jsx';
+const Portfolio = (props) => {
+
+  return (
+    <div className='portfolio-container'>
+      <div className='portfolio-primary-header'>
```

Comment:
Shouldn't the primary header be an h1? 

With css selectors you could have different css which looks like 

`.portfolio h1 {}`
`.portfolio h3 {}`

Or use a descendant selector 
`.portfolio-container > h3 {}`